gslib/utils/copy_helper.py

```diff
@@ -3687,11 +3689,12 @@ def _WriteRowToManifestFile(self, url):
         '%sZ' % row_item['end_time'].isoformat(),
         row_item['md5'] if 'md5' in row_item else '',
         row_item['upload_id'] if 'upload_id' in row_item else '',
-        row_item['size'] if 'size' in row_item else '',
-        row_item['bytes'] if 'bytes' in row_item else '',
+        str(row_item['size']) if 'size' in row_item else '',
+        str(row_item['bytes']) if 'bytes' in row_item else '',
         row_item['result'],
         row_item['description']]
-    data = [six.ensure_str(str(item).encode(UTF8)) for item in data]
+
+    data = [six.ensure_str(header) for header in data]
```

Comment:
Super nit: I think the name "header" here is misleading.  It will only be header in the case of the first row :)

Reply:
True! maybe 'result' or 'value'? I'll patch in a rename :)